Diferencia_Medias.py

```diff
@@ -2,10 +2,88 @@
 from Constantes import *
 import math
 import os
+
 """
     <summary>
         Funcion menu que realiza la prueba de hipotesis de una diferencia de medias.
     </summary>
+    <param name="x_bar1">Media de la población 1.</param>
+    <param name="x_bar2">Media de la población 2.</param>
+    <param name="sigma1">Desviación estándar de la población 1.</param>
+    <param name="sigma2">Desviación estándar de la población 2.</param>
+    <param name="n1">Numero de ensayos en la población 1.</param>
+    <param name="n2">Numero de ensayos en la población 2.</param>
+    <param name="delta">Diferencia proporcional que existe entre las medias.</param>
+    <param name="alpha">Nivel de significacia.</param>
+    <param name="Z">Valor del estadistico de prueba.</param>
```

Comment:
```suggestion
    <param name="z">Valor del estadistico de prueba.</param>
```

Tiene que ser exactamente igual que en el codigo

Reply:
Salv, se me fue ese xd